build.sbt

```diff
@@ -1,22 +1,22 @@
+import sbtorgpolicies.OrgPoliciesPlugin.autoImport.%%
+
 pgpPassphrase := Some(getEnvVar("PGP_PASSPHRASE").getOrElse("").toCharArray)
 pgpPublicRing := file(s"$gpgFolder/pubring.gpg")
 pgpSecretRing := file(s"$gpgFolder/secring.gpg")
 
 resolvers += Resolver.sonatypeRepo("snapshots")
+resolvers += Resolver.sonatypeRepo("releases")
+
+addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)
```

Comment:
Are we sure we need this compiler plugin in this place?

Reply:
I tried to put in the file `plugins.sbt` but it didn't work. After discuss it with @fedefernandez, he told me that the place for that line is `build.sbt`.